mypy.ini

```diff
@@ -30,3 +31,9 @@ ignore_missing_imports = True
 
 [mypy-pandas.*]
 ignore_missing_imports = True
+
+[mypy-django.*]
+ignore_missing_imports = True
+
+[mypy-rich.*]
```

Comment:
I am pretty sure that rich can handle types.

Reply:
yes, I think this is an error in the newest version of `mypy`. as same as `django-stub`, I will add `rich-stub` to be installed before running `mypy-precommit`